orangecontrib/infrared/widgets/owpreproc.py

```diff
@@ -730,6 +736,12 @@ def __init__(self, parent=None, **kwargs):
         self.lower = 0
         self.upper = 4000
         self.limits = 0
+        self.attrs = ['None']
```

Comment:
That would be cleaner, yes. As I said above, I put it so that there was something in the combobox...